mechanics/WORKING-GROUP-PROCESSES.md

```diff
@@ -42,9 +42,18 @@ the number of working groups low in order to keep things manageable.
 
 ## Proposing a new working group
 
-If you’ve identified a substantial architectural area which would benefit from
-long-lived, concerted and focused design, then you should consider creating a
-new working group. To do so, you need to:
+Generally, working groups should be founded once sufficient community interest
```

Comment:
Not sure if this is the right PR or the right doc - but I think it would be good to encourage this process over going off and solving something with the intent of trying to donate/merge it in later.

Reply:
I added a paragraph about this later.